spec/acceptance/suites/default/10_server_client_spec.rb

```diff
@@ -8,6 +8,7 @@
       skip('You need at least two hosts in your nodeset to run this test')
     end
   else
+    # test interoperability between 1st and 2nd server in the node set
     server1 = hosts[0]
```

Comment:
This needs to loop through all hosts and test the interconnections between each pair to make sure all the connections function properly.